src/nodes/experimental/NormalForm.jl

```diff
@@ -0,0 +1,176 @@
+@doc doc"""
+```Julia
+NormalForm(P, Q, V, Bᵤ, Cᵤ, Gᵤ, Hᵤ, Bₓ, Cₓ, Gₓ, Hₓ, Y_n, x_dims)
+```
+
+An implementation of the normal form for grid-forming actors following the publication:
```

Comment:
Maybe it would be good to explain why Au and Ax do not show up in the implementation in PowerDynamics. I think just writing one sentence would be enough.

Reply:
I added a note on this.